README.md

```diff
@@ -1,26 +1,90 @@
 [![Build Status](https://travis-ci.org/fossasia/open-event-ios.svg?branch=development)](https://travis-ci.org/fossasia/open-event-ios)
 
-# open-event-ios
+# Open Event iOS
 **iOS** app for **Open Event**
+![Open Event](https://storage.googleapis.com/eventyay.com/assets/branding/base_branding.png)
 
 ## Introduction
-> **This is an iOS app developed for `FOSSASIA` in mind.**
+> This is an iOS app developed for `FOSSASIA` in mind. The Open Event Project offers event managers a platform to organize all kinds of events including concerts, conferences, summits and regular meetups.
+
+## Communication
+
+**Please join our mailing list to discuss questions regarding the project :-**
+
+> *https://groups.google.com/forum/#!forum/open-event*
+
+**Our chat channel is here :-**
+
+> *https://gitter.im/fossasia/open-event*
 
 ## Things To Have
-1. **Xcode**
+1. **[Xcode](https://developer.apple.com/xcode/)**
 2. **[CocoaPods](http://cocoapods.org/)**
 
-## Setting Up
-1. Clone this repository
-2. `cd` to this repository
-3. Run `pod install`
-4. Open `.xcworkspace` file *(This is **important**)*
+## Development Setup
+
+Before you begin, you should already have the Xcode downloaded and set up correctly. You can find a guide on how to do this here: [Setting up Xcode](https://developer.apple.com/library/content/documentation/IDEs/Conceptual/AppStoreDistributionTutorial/Setup/Setup.html)
+
+##### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Steps to install Cocoapods (one time installation)
+
+- Run `sudo gem install cocoapods` to install the latest version of cocoapods
 
+-  Next, run `pod setup` for setting up cocoapods master repo. You may include `--verbose` for more descriptive logs.
+**NOTE:** This might take a while to setup depending on your network speed.
+
+## Setting up the iOS Project
+
+1. Download the _open_event_ios_ project source. You can do this either by forking and cloning the repository (recommended if you plan on pushing changes) or by downloading it as a ZIP file and extracting it.
+
+2. Navigate to the unzipped folder and run `pod install`.
+
+3. Open `FOSSAsia.xcworkspace` from the folder.
+
+4. Build the project (⌘+B) and check for any errors.
+
+5. Run the app (⌘+R).and test it.
 # Contribute!
 
 > You're now ready to contribute!
 
 Before writing any code, We will highly recommend to have a look at the [Contribution Guidelines](CONTRIBUTING.md) to avoid any unnecessary conflicts.
 
+## Git commands for Configuring remotes
```

Comment:
I think it is not a good idea to include all the git related instructions on Readme file.
- Make a **doc** folder. (As we have in SUSI iOS)
- create `gitconfiguraion.md` file in the doc folder.
- Put this section into gitconfiguration.md file.
- Give link to gitconfiguration in Readme.

Reply:
okk @imjog !